### DISCUSSION TITLE
Why zero values not NULL?

### DISCUSSION THREAD

**Discussion by frederikhors** [uptrace/bun#285] [Q&A]
Coming from `go-pg` I've always had zero values saved as `NULL` in postgres.

Can I ask you why did you decide to change this in Bun?

Is there an advantage in some sense? Performances? Disk space?

And how do I filter now? Example: if I have a `time.Time` column now the value in DB is `0001-01-01 00:00:00+00` instead of `NULL`.

Before I was using something like `where date != NULL`, what to do now? Maybe `where date > 0001-01-01 00:00:00+00`?

**Comment by vmihailenco** [comment]
Because it makes sense to disable the feature that some users don't want or don't expect.

See https://bun.uptrace.dev/guide/models.html#modeling-null-values

**Reply by frederikhors** [reply]
My question was to understand what has changed to make you change your mind from `go-pg` to `bun` about zero values to `null`?
